ast/policy.go

```diff
@@ -1153,14 +1161,18 @@ func (expr *Expr) Compare(other *Expr) int {
 		if cmp := Compare(t, other.Terms.(*SomeDecl)); cmp != 0 {
 			return cmp
 		}
+	case *Every:
+		if cmp := Compare(t, other.Terms.(*Every)); cmp != 0 {
+			return cmp
+		}
 	}
 
 	return withSliceCompare(expr.With, other.With)
 }
 
 func (expr *Expr) sortOrder() int {
 	switch expr.Terms.(type) {
-	case *SomeDecl:
+	case *SomeDecl, *Every: // TODO(sr): think about this more
```

Comment:
I think this needs to change... expr.sortOrder() is called above in Compare(). If the sortOrder is the same, the expr.Terms switch statement could be reached and the type assertions in the SomeDecl/Every cases will panic...